examples/inverse/plot_dics_beamformer.py

```diff
@@ -53,7 +53,8 @@
 evoked = epochs.average()
 
 # Read forward operator
-forward = mne.read_forward_solution(fname_fwd, surf_ori=True)
+forward = mne.read_forward_solution(fname_fwd)
+forward = mne.convert_forward_solution(forward, surf_ori=True)
```

Comment:
I think that all examples will work with `use_cps=True`, there are some tests which require `use_cps=False` while generating a fixed forward.